util_windows.go

```diff
@@ -0,0 +1,55 @@
+package pty
+
+import (
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+// InheritSize applies the terminal size of pty to tty. This should be run
+// in a signal handler for syscall.SIGWINCH to automatically resize the tty when
+// the pty receives a window size change notification.
+func InheritSize(pty Pty, tty Tty) error {
```

Comment:
Please keep generic code in "regular" files, and os specific one in build tag (or file suffix)  files.

Reply:
but they can only be called on windows, and already in util_windows.go